app/cypress/e2e/0-ndr-core-tests/gp_user_workflows/download_lloyd_george_workflow.cy.js

```diff
@@ -43,27 +84,116 @@ describe('GP Workflow: View Lloyd George record', () => {
                     body: baseUrl + '/browserconfig.xml', // uses public served file in place of a ZIP file
                 }).as('documentManifest');
 
+                cy.intercept('GET', '/SearchDocumentReferences*', {
+                    statusCode: 200,
+                    body: [
+                        {
+                            fileName: '1of2_testy_test.pdf',
+                            created: '2024-05-07T14:52:00.827602Z',
+                            virusScannerResult: 'Clean',
+                            id: 'test-id',
+                        },
+                        {
+                            fileName: '2of2_testy_test.pdf',
+                            created: '2024-05-07T14:52:00.827602Z',
+                            virusScannerResult: 'Clean',
+                            id: 'test-id-2',
+                        },
+                        {
+                            fileName: '3of2_testy_test.pdf',
+                            created: '2024-05-07T14:52:00.827602Z',
+                            virusScannerResult: 'Clean',
+                            id: 'test-id-3',
+                        },
+                    ],
+                }).as('searchDocumentReferences');
+
                 cy.getByTestId('download-all-files-link').should('exist');
                 cy.getByTestId('download-all-files-link').click();
-                cy.title().should('eq', 'Downloading documents - Digital Lloyd George records');
 
-                cy.wait('@documentManifest');
+                // Select documents page
+                cy.title().should(
+                    'eq',
+                    'Download the Lloyd George record for this patient - Digital Lloyd George records',
+                );
+                cy.wait('@searchDocumentReferences');
+
+                cy.getByTestId('patient-summary').should('exist');
+
+                cy.getByTestId('available-files-table-title').should('exist');
+                cy.getByTestId('download-selected-files-btn').should('exist');
+                cy.getByTestId('download-all-files-btn').should('exist');
+
+                cy.getByTestId('start-again-link').should('exist');
+                cy.getByTestId('download-all-files-btn').click();
+
+                cy.title().should('eq', 'Downloading documents - Digital Lloyd George records');
 
                 // Assert contents of page when downloading
                 cy.contains('Downloading documents').should('be.visible');
+                cy.contains(`Preparing download for 3 file(s)`).should('be.visible');
```

Comment:
If we want to check for these statements being present, can we use a dataId check rather than a text comparison?